synfig-studio/src/gui/duckmatic.cpp

```diff
@@ -606,7 +606,18 @@ Duckmatic::update_ducks()
 			{
 				synfig::Real radius = 0.0;
 				synfig::Point point(0.0, 0.0);
-				ValueNode_BLine::Handle bline(ValueNode_BLine::Handle::cast_dynamic(bline_vertex->get_link("bline")));
+				ValueNode_BLine::Handle bline;
```

Comment:
Maybe extract this as a method?
Also similar code in duck.cpp:291
```
ValueNode_BLine::Handle bline = ValueNode_BLine::Handle::cast_dynamic(bline_vertex->get_link("bline"));
```

Reply:
I can't see where I could place such method. :/
I'll implement another way

Reply:
How about something like this?
https://github.com/synfig/synfig/commit/0d5287a235a055774982d70bb14e29919460021f

Reply:
Wow! Got it! :D

Reply:
Done!

Reply:
Can I squash commits (for easy porting)?

Reply:
Sure